From a15e1e14d448af6f4790d62a50eea41a07cfa394 Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Wed, 18 Aug 2021 14:13:34 -0400
Subject: [PATCH 1/2] source/userint/HELPFRD.c: move qepcad.help to
 share/qepcad.

Instead of $qe/bin/qepcad.help, this file is much more appropriate in
$qe/share/qepcad/share, assuming $qe is usr-ish (in SageMath, $qe is
set to $SAGE_LOCAL).
---
 source/userint/HELPFRD.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/userint/HELPFRD.c b/source/userint/HELPFRD.c
index c7df76d..a60cf93 100644
--- a/source/userint/HELPFRD.c
+++ b/source/userint/HELPFRD.c
@@ -33,7 +33,7 @@ Step1: /* Open the text file containing the helps. */
           strcpy(helppath,HELPPATH);
        else {
           strcpy(helppath,qepath);
-          strcat(helppath,"/bin/qepcad.help"); }
+          strcat(helppath,"/share/qepcad/qepcad.help"); }
        if (!(fp = fopen(helppath,"r"))) {
          fprintf(stderr,"Error HELPFRD: Could not open %s\n",helppath);
          exit(1); }
-- 
2.31.1

